tools/spack-scripting/scripting/cmd/devbuildcosmo.py

```diff
@@ -11,6 +11,9 @@
 import llnl.util.tty as tty
 import re
 
+# same used by spack, comes with it
+import ruamel.yaml as yaml
```

Comment:
```suggestion
from ruamel import yaml
```
minor python issue, but I think we can remove this import anyways, since we can use `Spec.from_yaml`...